client/src/routes/Session/Session.tsx

```diff
@@ -122,9 +123,10 @@ const Session = () => {
 
   useEffect(() => {
     if (session?.ended) {
-      navigate('OutroPortal');
+      leaveSession();
+      navigate('OutroPortal', {portal: exercise?.outroPortal});
```

Comment:
Can't we leave this as it was, but just add a `leaveMeeting`?

Reply:
if you leave the meeting you don't have a session state anymore, and `OutroPortal` needs that data to be able to access the `outroPortal` from the session's exercise.

That's why I decided to pass in the Portal in the route params

Reply:
Yeah exactly, but if we don't use `leaveSession` here and only use `leaveMeeting` we will keep the session state while being in the outro portal too :)

Reply:
and then outro portal would call `leaveSession`, which would call `leaveMeeting`again, even though the user already left? 🤔

Reply:
or just call `resetSession` manually I guess?

Reply:
would be nice to always use the leaveSession hook in the same way when leaving 🤔

Reply:
True true... But I think you can run `leaveMeeting` multiple times, even though it's already left. Not sure what the best solution would be here tbh.